binderhub/builder.py

```diff
@@ -226,6 +242,7 @@ def escape(s):
             docker_host=self.settings['build_docker_host']
         )
 
+        """----- In Progress Builds -----"""
```

Comment:
There's no code about finding in progress builds here, they're onlin in build.py.